dotnet/greeter/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Net;
+
+namespace GreeterApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string host = "localhost";

```

Comment:
Does this work when we deploy as a container?
I think we need to bind IP to `0.0.0.0` instead of `localhost` to allow external communication.